core/test/addrs_test.go

```diff
@@ -0,0 +1,46 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGenerateTestAddrs(t *testing.T) {
+	var tableTest = []struct {
+		description    string
+		inputInt       int
+		expectedOutput []string
+	}{
+		{
+			description:    "generate 0 results",
+			inputInt:       0,
+			expectedOutput: []string{},
+		},
+		{
+			description: "generate a few results",
+			inputInt:    3,
+			expectedOutput: []string{
+				"/ip4/1.2.3.4/tcp/0",
+				"/ip4/1.2.3.4/tcp/1",
+				"/ip4/1.2.3.4/tcp/2",
```

Comment:
This assertion doesn't really make sense. The function would still be valid if the port numbers were different.